Sources/BinaryJSON/Context.swift

```diff
@@ -93,21 +93,21 @@ public protocol BitMaskOption: RawRepresentable {
     static func optionsBitmask(options: [Self]) -> Self.RawValue
 }
 
-public extension BitMaskOption where Self.RawValue: IntegerType {
+public extension BitMaskOption where Self.RawValue: Integer {
     
-    static func optionsBitmask<S: SequenceType where S.Generator.Element == Self>(options: S) -> Self.RawValue {
+    static func optionsBitmask<S: Sequence where S.Iterator.Element == Self>(options: S) -> Self.RawValue {
         return options.reduce(0) { mask, option in
             mask | option.rawValue
         }
     }
 }
 
-public extension SequenceType where Self.Generator.Element: BitMaskOption, Self.Generator.Element.RawValue: IntegerType {
+public extension Sequence where Self.Iterator.Element: BitMaskOption, Self.Iterator.Element.RawValue: Integer {
     
-    func optionsBitmask() -> Self.Generator.Element.RawValue {
+    func optionsBitmask() -> Self.Iterator.Element.RawValue {
         
         let array = self.filter { (_) -> Bool in return true }
         
-        return Self.Generator.Element.optionsBitmask(array)
+        return Self.Iterator.Element.optionsBitmask(array)
     }
 }
```

Comment:
Could you do me favor and insert a new line at the end of the file? I just need you to push a commit to trigger travis one last time.